github4s/src/main/scala/github4s/algebras/Repositories.scala

```diff
@@ -271,6 +271,21 @@ trait Repositories[F[_]] {
       headers: Map[String, String] = Map()
   ): F[GHResponse[List[User]]]
 
+  /**
+   * List of releases
+   *
+   * @param owner of the repo
+   * @param repo name of the repo
+   * @param headers optional user headers to include in the request
```

Comment:
:art: could you add scaladoc for the pagination param?

Reply:
Thank you very much!
 I have just added it.